### PR TITLE
Update closure-util to the latest version

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by fredj** [comment]
I've an issue (locally) with the `transpile.js` (file not found)

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "async": "2.5.0",
     "bootstrap": "3.3.7",
     "clean-css-cli": "4.1.10",
-    "closure-util": "camptocamp/closure-util#487fac6",
+    "@camptocamp/closure-util": "camptocamp/closure-util#f1152e1",
```
Comment (sbrunner) [inline package.json]: Can't we do a release on npm?

Comment (fredj) [reply package.json]: You're right, version 1.24.0 is already published on npm

Comment (sbrunner) [reply package.json]: :-)

**Comment by fredj** [comment]
:tada:

**Pull request merged**